README.md

```diff
@@ -49,9 +49,9 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.8.0 \
-       --bootstrap kubeadm:v1.8.0 \
-       --control-plane kubeadm:v1.8.0 -v5
+   clusterctl init --core cluster-api:v1.8.1 \
+       --bootstrap kubeadm:v1.8.1 \
+       --control-plane kubeadm:v1.8.1 -v5
```

Comment:
I think we decided not to touch these README for patch releases?

Reply:
I do not remember. But why? 
IMHO if we do not change every time we most likely forget to update it.

Reply:
It prevents dependabot from doing these bumps as it cannot handle README.md

README doesn't need to update all the time, so bump once when bumpign minor and then leave alone and let bot do it.

Reply:
noted. Next time I will not update it.